torch/_inductor/scheduler.py

```diff
@@ -820,6 +820,15 @@ def get_estimated_runtime(self) -> float:
         try:
             gpu_memory_bandwidth = get_gpu_dram_gbps()
             gpu_flops = get_device_tflops(dtype) * 10**12
+            # If cudaGetDeviceProperties returns 0 for gpu_memory_bandwidth or gpu_flops
+            # there is a chance to continue execution successfully. Otherwise, it would fail with
+            # ZeroDivisionError below.
+            if gpu_memory_bandwidth <= 0:
+                raise ValueError(
```

Comment:
Probably should raise AssertionError as ValueError usually denotes user provided input, not violated invariants in the output